assets/src/scss/new-identity/style.scss

```diff
@@ -121,18 +121,41 @@ table.spreadsheet-table.is-color-gp-green {
   }
 }
 
-a:where(.standalone-link) {
+a:where(.standalone-link, .split-two-column-item-link) {
+  --box-shadow--offset-inset: inset;
```

Comment:
I've added this local CSS Variables to avoid duplicating properties and make it easy to follow up.